amr-wind/wind_energy/ABLStats.cpp

```diff
@@ -441,34 +437,17 @@ void ABLStats::write_netcdf()
             }
         }
 
-        {
-            amrex::Vector<std::string> var_names{
-                "u'v'_sfs", "u'w'_sfs", "v'w'_sfs"};
-            for (int i = 0; i < AMREX_SPACEDIM; i++) {
-                m_pa_sfs->line_average(i, l_vec);
-                auto var = grp.var(var_names[i]);
-                var.put(l_vec.data(), start, count);
-            }
-        }
-
-        {
-            amrex::Vector<std::string> var_names{
-                "u'theta'_sfs", "v'theta'_sfs", "w'theta'_sfs"};
-            for (int i = 0; i < AMREX_SPACEDIM; i++) {
-                m_pa_tsfs->line_average(i, l_vec);
-                auto var = grp.var(var_names[i]);
-                var.put(l_vec.data(), start, count);
-            }
-        }
+        output_sfs_stress_avgs(grp, nt);
     }
     ncf.close();
 #endif
 }
 
-void ABLStats::calc_sfs_stress_avgs()
+#ifdef AMR_WIND_USE_NETCDF
+void ABLStats::output_sfs_stress_avgs(ncutils::NCGroup grp, const size_t nt)
```

Comment:
We can make this a standalone function

Reply:
Addressed in https://github.com/Exawind/amr-wind/pull/290/commits/da05df00d9686f43b32742e3ec13693811db1ab3